x-pack/plugins/apm/public/components/app/Settings/agent_keys/create_agent_key/agent_key_callout.tsx

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { Fragment } from 'react';
+import { i18n } from '@kbn/i18n';
+import {
+  EuiSpacer,
+  EuiCallOut,
+  EuiButtonIcon,
+  EuiCopy,
+  EuiFormControlLayout,
+} from '@elastic/eui';
+
+interface Props {
+  name: string;
+  token: string;
+}
+
+export function AgentKeyCallOut({ name, token }: Props) {
+  return (
+    <Fragment>
+      <EuiCallOut
```

Comment:
In the event that the user deletes the key, the callout will continue to display. Not sure if that's a common event, but I think it would be nice to ensure that the view refreshes if there's a change in the table.

<img width="1181" alt="CleanShot 2021-12-06 at 09 42 05@2x" src="https://user-images.githubusercontent.com/4104278/144838019-ce320caa-30b8-4062-a910-f539de1db8c5.png">

Reply:
It makes sense, thanks. I've changed this so only the button in the prompt is visible in this case